internal/cmd/cluster/ls.go

```diff
@@ -51,7 +47,7 @@ func render(it cluster.Iterator, consume func(routing.Record) error) error {
 }
 
 func formatter(c *cli.Context) func(routing.Record) error {
-	if c.Bool("json") {
+	if c.Bool(_json) {
```

Comment:
I think we can stick with the string-literal `"json"` instead.  I don't think there's much to be gained from defining a constant.